### PR TITLE
real cosmos

### PR DESCRIPTION
**Pull request by r10s**
by the wording "cosmos" i was insipired by sth else than a grey background :)

also, this makes the table nicer browsable on desktop.

see screenshots for before/after, for a local preview, just checkout this branch and open index.html locally.

there are more things to tweak, but before doing so, there should be some consent that we want to go this way.

Screenshot 2022-12-16 at 20 54 12
Screenshot 2022-12-16 at 20 54 16

... the starfield is animated, of course :) EDIT: the screenshot is outdated, check out  index.html locally.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,42 +217,51 @@ <h1 class="title">
...
 </tr>
 
-</table>
+<tr>
+  <th colspan="3">Misc.</th>
+</tr>
+<tr>
+  <td colspan="3">
+    <a href="https://chaos.social/@delta">Mastodon</a>,
+    <a href="https://twitter.com/delta_chat">Twitter</a>,
```
Comment (adbenitez) [inline index.html:244]: twitter is being removed from other pages, maybe also from here?

**Review by adbenitez — APPROVED** [review]
now this is a more immersive experience :sunglasses:  
nice!

**Review by rosano — COMMENTED** [review]
Looks fun! I generally avoid light text on dark background for readability, but I think it's safe to go full nerd here.

I prefer lower case on the first columns (except for the names). I would use a smaller font for the second and especially third columns as they're meant more to be clicked than read, and it can save some space.

The centering/italics for video links looks to me visually as if it's a note or statement about the previous content. I think the original had some issues too and we should experiment to find something better, but I'd lean towards normal alignment and style.

You might like to style this page's cosmos banner to match the new style.

# Info container

I feel it's a little too wide (too much space before and after column 3), would cap it at something like 560px. Testing with responsive mode in Firefox/Safari, it seems iPhone 5 and 6 screens scroll horizontally; on mobile, I would also shorten the top margin or remove it.

# Title

Screen Shot 2022-12-17 at 10 04 48

I think the logo is awkwardly small here and would make it at least slightly larger than the text, maybe 32-36px. The text is probably readable for most people, but a little thin; you might want to experiment with font-weight 200-400 for Helvetica Neue, but not sure if it will apply to any fallbacks. 


# Headings

Screen Shot 2022-12-17 at 10 05 07
Screen Shot 2022-12-17 at 10 05 20

I would restore the original heading style of all caps + letter spacing to contrast more with the rest of the text. It's nice to have a thin line as decoration but for the header row it feels unnecessary; creates 'edges' in a context that's quite infinite and 'space'-oriented. I would visually differentiate headers that are clickable and/or use a different color for the row links.

# Misc.

Screen Shot 2022-12-17 at 10 05 34

Love how compact this section is now, great improvement. Maybe experiment with:
- centering
- using CSS margin or pipeline characters to separate the links
- an alternate text style

**Comment by hpk42** [comment]
would maybe add simplebot links and asyncpython bindings. otherwise fine with it if rosano is fine with it :)

**Comment by r10s** [comment]
thanks for the detailed feedback, i'd tried to incorporate most of it.

i think, this looks even more "spacy" now :)

some comments:

- i'd like to use the logo variation without the white border, i think, this will look better. will do a commit.
- i also think, the column widths can still be optimised, however, honestly, this was also not perfect before, esp. on desktop
- yip, i also thought the banner should be black now

**Comment by r10s** [comment]
an updated screenshot below; i think, this is mergable now :)

Screenshot 2022-12-17 at 22 17 56

File: index.html [hunk 2]
```diff
@@ -9,59 +9,65 @@
...
-  margin-top: 0;
-
-  background: linear-gradient(120deg, rgb(119, 136, 143), rgb(54, 78, 89));
+  margin: 0px;
+  padding: 0px;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  background-color: #000;
+  color: #bbb;
+  font-size: 14px;
+  font-weight: light;
```
Comment (rosano) [inline index.html:24]:
```suggestion
  font-weight: lighter;
```

I think this is a typo

File: index.html [hunk 3]
```diff
@@ -207,56 +210,70 @@ <h1 class="title">
...
 </tr>
 <tr>
   <td>Autocrypt</td>
   <td><a href="https://autocrypt.org/">autocrypt.org</a></td>
   <td><a href="https://github.com/autocrypt/autocrypt">autocrypt/autocrypt</a></td>
 </tr>
 <tr>
-  <td>verified contacts/groups</td>
-  <td><a href="https://countermitm.readthedocs.io/">countermitm.readthedocs.io</a></td>
+  <td>Verification</td>
```
Comment (rosano) [inline index.html:240]:
```suggestion
  <td>verification</td>
```

If it's not name, maybe lower case?

File: index.html [hunk 4]
```diff
@@ -207,56 +210,70 @@ <h1 class="title">
...
 <tr>
-  <td>others</td>
+  <td>Other</td>
   <td></td>
   <td><a href="https://github.com/deltachat/deltachat-core-rust/blob/master/standards.md">standards.md</a></td>
 </tr>
 
-</table>
+<tr>
+  <th colspan="3">Misc.</th>
```
Comment (rosano) [inline index.html:251]:
```suggestion
  <th colspan="3">Etcetera</th>
```

Maybe we could use the unabbreviated form, or another word?

File: index.html [hunk 5]
```diff
@@ -131,34 +134,34 @@ <h1 class="title">
...
   <td><a href="https://bots.delta.chat">bots.delta.chat</a></td>
   <td><a href="https://github.com/deltachat-bot/bot-pages">deltachat-bot/bot-pages</a></td>
 </tr>
 <tr>
-  <td>echo bot</td>
-  <td></td>
-  <td><a href="https://github.com/deltachat-bot/echo">deltachat-bot/echo</a></td>
+  <td>examples</td>
+  <td><a href="https://github.com/deltachat-bot/echo">echo</a></td>
+  <td><a href="https://support.delta.chat/c/bots/">all</a></td>
```
Comment (rosano) [inline index.html:159]:
```suggestion
  <td><a href="https://support.delta.chat/c/bots/">more bots</a></td>
```

I try to use unique link text for accessibility reasons.

File: index.html [hunk 6]
```diff
@@ -131,34 +134,34 @@ <h1 class="title">
...
 <tr>
-  <td colspan="3"><a href="https://github.com/orgs/webxdc/repositories?type=all">existing apps</a></td>
-</tr>
-<tr>
-  <td>hello world</td>
-  <td></td>
-  <td><a href="https://github.com/webxdc/hello">webxdc/hello</a></td>
+  <td>examples</td>
+  <td><a href="https://github.com/webxdc/hello">hello world</a></td>
+  <td><a href="https://github.com/orgs/webxdc/repositories?type=all">all</a></td>
```
Comment (rosano) [inline index.html:139]:
```suggestion
  <td><a href="https://github.com/orgs/webxdc/repositories?type=all">more webxdc</a></td>
```

I try to use unique link text for accessibility reasons.

File: index.html [hunk 7]
```diff
@@ -207,56 +210,70 @@ <h1 class="title">
...
+  <th colspan="3">Misc.</th>
+</tr>
+<tr>
+  <td colspan="3" align="center">
+    <a href="https://chaos.social/@delta">Mastodon</a> &middot;
+    <a href="https://www.youtube.com/@deltachat/videos">YouTube</a> &middot;
+    <a href="https://support.delta.chat">Forum</a> &middot;
+    <a href="https://web.libera.chat/#deltachat">IRC</a> &middot;
+    <a href="https://www.transifex.com/delta-chat/public/">Translation</a> &middot;
+    <a href="https://delta.chat/en/contribute#donate-money">Donations</a> &middot;
```
Comment (rosano) [inline index.html:260]:
```suggestion
    <a href="https://delta.chat/en/contribute#donate-money">Donate</a> &middot;
```

Can be shorter?

**Review by rosano — COMMENTED** [review]
I think this looks much better. Two issues I think are important are:

1) It still seems to scroll horizontally on mobile. I would try fitting to the device width using something like:

```css
@media (max-width: 450px) {
	#info {
		width: 100%;
	}
}
```
2) It should be possible to visually discover the header links without hovering, I would use some kind of light underline for clickable header links.

And a minor thing, I think I spelled it as DeltaChat but should we change it to two words?

If you feel like moving on, I can make my own pull request with these changes later.

**Comment by rosano** [comment]
Seems like the mobile scrolling issue was on the original too so maybe I can tackle it myself.

File: starfield.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+
+//	Define the starfield class.
+function Starfield() {
+	this.fps = 30;
+	this.canvas = null;
+	this.width = 0;
+	this.height = 0;
+	this.minVelocity = 15;
+	this.maxVelocity = 30;
+	this.stars = 100;
```
Comment (rosano) [inline starfield.js:14]:
```suggestion
	this.stars = 200;
```

I could imagine a few more stars.

**Comment by r10s** [comment]
> It should be possible to visually discover the header links without hovering, I would use some kind of light underline for clickable header links.

i also do not like that part. however, as different header layout do not look great as well, i would suggest to not use clickable headers at all and to move the few links somehow to the other "link fields" below.

File: starfield.js (hunk 8 above)
Comment (r10s) [reply starfield.js:14]: this does not look good on small screens / mobile.

so, unless we adapt that somehow, i suggest to stay with the current default - that is "starry" on mobile and "decent" on desktop.

File: index.html (hunk 2 above)
Comment (r10s) [reply index.html:24]: i thought `light` exists as well, but you're right, [it is just `lighter`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) :)

File: index.html (hunk 4 above)
Comment (r10s) [reply index.html:251]: at least in my world, as a non-native english, "Etcetera" is much more weird as "Misc." (although, "Etc." is known of course :) i also think, ppl will stumble upon "Miscellaneous" more than over "Misc.".

maybe "Various"?

Comment (rosano) [reply index.html:251]: > maybe "Various"?

Sure, or 'Elsewhere' or 'On the web'. But not a big deal: I just see the abbreviation dot in the context of the dot separators and it makes me want to remove it.

**Review by rosano — APPROVED** [review]

**Comment by r10s** [comment]
let's merge that in and refine in subsequent pr, after getting some experience :)

**Pull request merged**